Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+        private const string InValidOutput = "N/A.";
+        private readonly StringBuilder _persons;
+        private readonly decimal _amount;
+        public const string culture = "en-GB";
+
+        public BalanceStats(StringBuilder persons, decimal amount)
+        {
+            _persons = persons;
+            _amount = amount;
+        }
+
+        public StringBuilder GetPersons()
+        {
+            return _persons;
+        }
+
+        public decimal GetAmount()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetAmount' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)